src/utils.js

```diff
@@ -497,3 +497,17 @@ export const md5Hash = item => {
   md5sum.update(JSON.stringify(item));
   return md5sum.digest('hex');
 };
+
+export const isEncodingType = encodingType =>
+  !!(
+    encodingType &&
+    typeof encodingType === 'string' &&
+    ['ascii', 'utf8', 'utf16le', 'ucs2', 'base64', 'binary', 'hex'].includes(
```

Comment:
there's also `utf-8`, `utf-16-le` etc.:
https://stackoverflow.com/questions/14551608/list-of-encodings-that-node-js-supports
maybe you can import this list from somewhere else?

Reply:
Its buffer encodings found this list here
https://stackoverflow.com/questions/34476754/what-encodings-does-buffer-tostring-support